tests/functional/test_task_samples.py

```diff
@@ -49,20 +50,41 @@ def _skip(self, validation_output):
         return False
 
     def test_task_samples_is_valid(self):
-        plugins.load()
         rally = utils.Rally(force_new_db=True)
+        # In TestTaskSamples, Rally API will be called directly (not via
+        # subprocess), so we need to change database options to temp database.
         db.db_options.set_defaults(
-            db.CONF, connection="sqlite:///%s/db" % rally.tmp_dir,
-            sqlite_db="rally.sqlite")
+            db.CONF, connection="sqlite:///%s/db" % rally.tmp_dir)
+
+        # let's use pre-created users to make TestTaskSamples quicker
+        deployment = api.Deployment.get("MAIN")
```

Comment:
I didn't run the test case before, could you please kindly point out where is the "MAIN" deployment created.